engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CachingSimpletonActor.scala

```diff
@@ -0,0 +1,47 @@
+package cromwell.engine.workflow.lifecycle.execution.callcaching
+
+import akka.actor.{Actor, ActorLogging, Props}
+import cromwell.core.JobOutputs
+import cromwell.database.CromwellDatabase
+import cromwell.database.sql.{CachedResult, MetaInfoId}
+import cromwell.engine.workflow.lifecycle.execution.callcaching.CachingSimpletonActor.{CachedOutputLookupSucceeded, CachedOutputLookupFailed, CacheResultResponse}
+import wdl4s.TaskOutput
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+object CachingSimpletonActor {
```

Comment:
I'd suggest a different name for this actor.  The fact that it has to manipulate WDL simpletons is an implementation detail invisible to the outside, and the name doesn't really say what this actor actually does (fetch results for a cache hit from the db)